src/main/kotlin/com/vannak/tech/api_project/domain/model/User.kt

```diff
@@ -8,13 +8,19 @@ import javax.persistence.*
 import kotlin.collections.ArrayList
 
 @Entity
+@Table(name = "users")
 data class User(
         @Id
-        @GeneratedValue(strategy = GenerationType.AUTO)
+        @GeneratedValue(strategy = GenerationType.TABLE)
+        @Column(name="id")
         var id:Long=0,
+        @Column(name = "name")
         var name:String,
+        @Column(name = "dob")
         var dob:Date,
+        @Column(name = "phone_number")
         var phoneNumber: String,
+        @Column(name = "email")
         var email:String,
```

Comment:
- Implement Time Stamp CreatedAt and UpdatedAt
- Column Role should be outside of parameter attribute of user.